source/funkin/game/PlayState.hx

```diff
@@ -1226,7 +1230,7 @@ class PlayState extends MusicBeatState
 
 	function updateRatingStuff() {
 		scoreTxt.text = 'Score:$songScore';
-		missesTxt.text = '${comboBreaks ? "Combo Breaks" : "Misses"}:$misses';
+		missesTxt.text = '${comboBreaks ? "Combo Breaks" : "Misses"}:${comboBreaks ? (breaks + misses) : misses}'; // yes, this is combo breaks work, if you don't believe me, heres proof, https://www.google.com/search?client=opera-gx&q=How+do+combo+breaks+work+in+rhythm+games%3F&sourceid=opera&ie=UTF-8&oe=UTF-8
```

Comment:
Bro uses opera gc 💀

Reply:
lol, I didn't realize that

Reply:
![image](https://github.com/user-attachments/assets/ee1d30d5-6763-46bc-aca2-d6dbf71339d6)
its supposed to be this

Reply:
thats just for Ensemble Stars, id probs recommend going by Osu! rules

Reply:
whats osus rules

Reply:
plus when doing this I just kinda went by most mods rules

Reply:
tho if there are any changes you would like to request I'd be happy to oblige